src/lsm/manifest_log.zig

```diff
@@ -115,6 +124,25 @@ pub fn ManifestLogType(comptime Storage: type) type {
 
         next_tick: Grid.NextTick = undefined,
 
+        /// A map from table address to the manifest block and entry that is the latest extent
+        /// version. Used to determine whether a table should be dropped in a compaction.
+        tables: Tables,
```

Comment:
Was tripped more than once by the naming here! This aren't tables actually, just extents. `extents: Extents` or `table_extents` would make more sense to me.